### PR TITLE
Remove Rust runtime dependency in the Exonum tests [ECR-3931] 

### PR DESCRIPTION
**Pull request by alekseysidorov**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -131,6 +131,7 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 #### exonum
 
 - `sandbox` module was moved to the `test-suite/consensus-tests`. (#1627)
+- Some of test was moved to the `test-suite/node-tests`. (#1633)
```
Comment (popzxc) [inline CHANGELOG.md]:
```suggestion
- Some of general-purpose tests were moved to the `test-suite/node-tests`. (#1633)
```

File: exonum/src/blockchain/tests.rs [hunk 2]
```diff
@@ -12,344 +12,340 @@
...
+    fn default() -> Self {
+        InitAction::Noop
+    }
+}
 
-        let instance_id = {
-            let mut instance_id_entry = context.service_data().get_entry("instance_ids");
-            let instance_id = instance_id_entry.get().unwrap_or(TEST_SERVICE_ID + 1);
-            instance_id_entry.set(instance_id + 1);
+/// Actions that performs at the `after_transactions` stage.
```
Comment (popzxc) [inline exonum/src/blockchain/tests.rs]:
```suggestion
/// Actions to be performed at the `after_transactions` stage.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -139,6 +139,7 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 #### exonum
 
 - `sandbox` module was moved to the `test-suite/consensus-tests`. (#1627)
+- Some of general-purpose tests were moved to the `test-suite/node-tests`. (#1633)
```
Comment (aleksuss) [inline CHANGELOG.md]: Split entities with empty line please

File: test-suite/node-tests/src/proto/services.proto [hunk 4]
```diff
@@ -14,22 +14,23 @@
...
+    exonum.crypto.PublicKey from = 1;
+    exonum.crypto.PublicKey to = 2;
+    uint64 amount = 3;
+  }
+  
+  message Issue {
+    exonum.crypto.PublicKey to = 2;
+    uint64 amount = 3;
+  }
+
```
Comment (aleksuss) [inline test-suite/node-tests/src/proto/services.proto]: New line ?

File: CHANGELOG.md [hunk 5]
```diff
@@ -128,6 +128,7 @@ Indexes iterators names has been shortened to `Iter`, `Keys` and `Values`. (#162
   in the state aggregation. (#1553)
 
 - Added support of unsized keys to `MapIndex`. (#1621)
```
Comment (aleksuss) [inline CHANGELOG.md]: Merge artifact ?

**Pull request merged**